src/main.cpp

```diff
@@ -71,7 +71,7 @@ void setup()
 
     SerialCommands::setUp();
 
-#if IMU == IMU_MPU6500 || IMU == IMU_MPU6050 || IMU == IMU_MPU9250 || IMU == IMU_BNO055 || IMU == IMU_ICM20948 || IMU == IMU_BMI160|| IMU == IMU_ICM42688
+#if IMU == IMU_MPU6500 || IMU == IMU_MPU6050 || IMU == IMU_MPU9250 || IMU == IMU_BNO055 || IMU == IMU_ICM20948 || IMU == IMU_BMI160 || IMU == IMU_BMI270 || IMU == IMU_ICM42688
```

Comment:
I think "blacklist" would work better. But then, which sensors didn't like that? BNO085? Maybe it was a problem when bno reset procedure was quite broken?

Reply:
The feature was added when no one had BNO. And i think no one was sure if it works with it or not.

Reply:
After more thinking and testing, I'm suggesting to enable bus reset for all sensors. From initial testing BNO doesn't care, similar to ICM42688.

Reply:
If there are any changes to BNO, the merge would have to wait until the next major version release, because we want to update firmware in the next batch of the official trackers.

Reply:
@l0ud see last comment of [Eirenliel](https://github.com/Eirenliel)

Reply:
According to Meia on Discord bmi160 is not affected with reset (quoted code is outdated, there is no black/whitelist in sfusion anymore).
According to... some other report on Discord BNO085 trackers with LSM6DSV as aux also work fine. Also in my testing done some time ago bno085 was still working fine.
I think it's good not to do any #ifdefs under the current circumstances.